Breadcrumbs/Breadcrumbs.vue

```diff
@@ -4,6 +4,17 @@ import { config } from '~/config'
 // Constants
 import { BUTTON_PRESET } from '~/components/Button/constants/button-preset.constant'
 
+const { notAdaptive = false, hideBtnPosition = 'left' } = defineProps<{
```

Comment:
Please stick to `withDefaults` syntax for now. Even though the experimental `propsDestructure` is currently turned on in the project, we should do it all at once when we're ready, not have multiple syntaxes all over the place